.github/workflows/ci.yml

```diff
@@ -255,6 +255,7 @@ jobs:
           token: ${{secrets.CODECOV_TOKEN}}
 
   build-and-upload:
+    if: github.event_name == 'push'
```

Comment:
I don't think that this will help? Since it still active on all PR branches that targets main?

Reply:
Or will the event_name be `pull_request` for all pushes to a PR branch?

Reply:
You mean all current branches that are open? We could rebase / cherry pick until they are all closed out?

Reply:
I meant for future branches too, the current branches I wouldn't worry about too much.
Let's merge this and see what happens. :smile:

Reply:
> Or will the event_name be `pull_request` for all pushes to a PR branch?

Ah yes, so from what I understand the CI triggers on PR's and Pushes to main. I believe a push to a feature branch that has an open PR triggers the the PR flow, not the Push flow.

Lets see, easy enough to revert :smile: